### ISSUE TITLE
bug(mat-select): Unable to navigate beyond first select option using iOS/Android screen reader

### ISSUE THREAD

**Issue by jlai** [angular/components#27690]
### Is this a regression?

- [X] Yes, this behavior used to work in the previous version

### The previous version in which this bug was not present was

15.x

### Description

I'm unable to select options other than the first options using using mat-select using a screen reader on mobile (desktop seems OK). This appears to be a regression since it occurs on the latest [material.angular.io examples](https://material.angular.io/components/select/examples) but not on the [v15 examples](https://v15.material.angular.io/components/select/examples).

After activating the selection field, it opens a menu, but swiping to the right the screen reader says that it's the last item and I'm unable to select the other options.

### Reproduction

Reproducible on Android 14 Talkback and iOS 15 Voiceover, but not on Mac OS Voiceover

Steps to reproduce:
1. Open https://material.angular.io/components/select/examples in Safari/Chrome on iOS or Android
2. Turn on Voiceover/Talkback
3. Go to simple example (mat-select rather than native, not disabled), such as the "select in a form" example
4. Double-tap to activate the select element. A menu should open.
5. Swipe right to try to navigate through the select menu

### Expected Behavior

Able to navigate through menu options and select items

### Actual Behavior

Unable to navigate past the first menu item using the screen reader

### Environment

- Angular: 16.2.1
- CDK/Material:
- Browser(s): Safari/Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): iOS and Android

**Comment by aHereitani** [comment]
Hi @zarend, 

are you still aware of this issue? 

MatSelect is unusable on mobile devices using Voiceover/Talkback. Therefore I consider the usage of MatSelect is prohibited in Software falling under ADA or EAA law. Maybe you can check, if this issue is tracked with the appropriate priority?

Regards,
Alex

**Comment by adolgachev** [comment]
This was caused by https://issues.chromium.org/issues/40260928 which was fixed with Chrome 137
